--- local.conf	2018-03-09 13:37:15.867054594 -0600
+++ local.conf	2018-03-10 09:23:39.000000000 -0600
@@ -1,10 +1,22 @@
-maxgputemp 85
+
+flags -colors 1 -tt 50 -tstop 85 -ttli 75 -r 1 -wd 1 -gser 1
+
 stratumproxy enabled
-proxywallet 0x0bdC4F12fB57d3acA9C3cF72B7AA2789A20d27f2
-proxypool1 pool-usa.ethosdistro.com:5001
-proxypool2 pool-eu.ethosdistro.com:5001
-flags --cl-global-work 8192 --farm-recheck 200
-globalfan 85
+custompanel PerlitPerlit
+
+#cor 113-58085SHD1-M80 1300
+
+#globalfan 92
+lockscreen enabled
+
+globalminer claymore
+proxywallet 0x8cf0bf2DDb22Ee20B9A3489C56203807d1285905
+
+proxypool2 eu1.ethermine.org:4444
+proxypool1 asia1.ethermine.org:4444
+
+#globalname disabled
+
 
 # ethOS Pool: ethosdistro.com/pool
 # QUICK START GUIDE: ethosdistro.com/Q
